meta-mender-imx6ul/templates/local.conf.append

```diff
@@ -0,0 +1,95 @@
+
+# Appended fragment from meta-mender-community/meta-mender-nxp/templates
+# Supported machine is:
+#    - imx6ulevk
+#
+
+# Use separate dir for downloads and cache (for speed up successive builds)
+# Replace '~/' with your 'home' directory (Yocto does not expand ~/)
+# DL_DIR = "~/.yocto/warrior/downloads/"
+# SSTATE_DIR = "~/.yocto/warrior/sstate-cache"
+
+# REQUIRED CONFIGURATION:
+# The name of the image or update that will be built
+# This is what the device will report that it is running
+# and different updates must have different names
+MENDER_ARTIFACT_NAME = "release-1"
+
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
+VIRTUAL-RUNTIME_initscripts = ""
```

Comment:
This is already provided by https://github.com/mendersoftware/meta-mender-community/blob/zeus/templates/local.conf.append. Think it is best to utilize that instead.